core/src/main/java/nl/nn/adapterframework/pipes/Json2XmlValidator.java

```diff
@@ -188,43 +189,44 @@ public PipeRunResult doPipe(Message input, PipeLineSession session, boolean resp
 			throw new PipeRunException(this, "Cannot align JSON", e);
 		}
 	}
-	
+
 	protected RootValidations getJsonRootValidations(boolean responseMode) {
 		if (isValidateJsonToRootElementOnly()) {
 			String root=getMessageRoot(responseMode);
 			if (StringUtils.isEmpty(root)) {
 				return null;
 			}
 			return new RootValidations(root);
-		} 
+		}
 		return getRootValidations(responseMode);
 	}
-	
+
 	protected PipeRunResult alignXml2Json(String messageToValidate, PipeLineSession session, boolean responseMode) throws XmlValidatorException, PipeRunException, ConfigurationException {
 
 		ValidationContext context = validator.createValidationContext(session, getJsonRootValidations(responseMode), getInvalidRootNamespaces());
 		ValidatorHandler validatorHandler = validator.getValidatorHandler(session,context);
-		
+
 		// Make sure to use Xerces' ValidatorHandlerImpl, otherwise casting below will fail.
 		XmlAligner aligner = new XmlAligner(validatorHandler);
+		//aligner.setIgnoreUndeclaredElements(isIgnoreUndeclaredElements()); // cannot ignore XML Schema Validation failure in this case, currently
```

Comment:
misschien is het handig voor de forward compatibiliteit om in de configure te zorgen dat je een warning/error krijgt wanneer je dit zet bij XML2JSON.